mirrord/cli/src/main.rs

```diff
@@ -9,8 +9,6 @@ use anyhow::{anyhow, Context, Result};
 use clap::Parser;
 use config::*;
 use const_random::const_random;
-#[cfg(all(target_os = "macos", target_arch = "aarch64"))]
-use errno;
```

Comment:
How does that work if we use `errno` there: https://github.com/metalbear-co/mirrord/blob/a76adadaca201b4e9408e85df2ad8bea10728987/mirrord/cli/src/main.rs#L234?

Reply:
`errno:Errno` is "fully qualified" name. In Rust `use errno` usually has no particular meaning